tools/protocol-schema-check/res/protocol_schema.toml

```diff
@@ -157,6 +157,7 @@ IgnoredVecU8 = 1855789801
 IntegerOverflowError = 2542362165
 InvalidAccessKeyError = 2954698659
 InvalidTxError = 1219344901
+InvalidWitnessesInfo = 2205399556
```

Comment:
Does this really need to be a part of a protocol schema? Yes, we do store it in DB, but even if the data in the DB gets invalidated here, it won't be able to affect anything more than just the dump invalid witnesses command which itself is a debug tool…